doc/basic_pv_pvc_dv.md

```diff
@@ -0,0 +1,219 @@
+# Storage
+
+Storage can be a tricky subject in kubernetes so we would like to give some basic guidance on how to implement storage for use with CDI. Storage in kubernetes is defined by Persistent Volumes ([PV](https://kubernetes.io/docs/concepts/storage/persistent-volumes/)) which are a unit of storage. PVs define their size and other information needed to define the storage. When a user wants to use storage they define a Persistent Volume Claim ([PVC](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims)), which specifies the size of storage they want, and some other optional information. Kubernetes then tries to match an available PV to the PVC. If one is found, then the PVC is bound to the PV and the PVC becomes available. A pod can now mount the PVC using the pod spec when it starts.
```

Comment:
I'd leave out the "tricky" part (even if it's somewhat true) and instead just provide a high level description of storage in Kubernetes, mostly rely on the upstream docs you already have linked.  Maybe break out this first section you have into some sub-sections... somethign like:

```
## Kubernetes Storage Overview
As a user of CDI it's important to have a clear understanding of the storage objects provided by Kubernetes.  Be sure you have a thorough understanding of [P(https://kubernetes.io/docs/concepts/storage/persistent-volumes/))ersistentVoumes[ and [PersistentVolumeClaims](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims)).  

## Types of Storage 
There are three main categories of provisioners available in Kubernetes:
1. Manual hostPath
2. Local Disk
3. Dynamic Provisioners
```